vllm/compilation/monitor.py

```diff
@@ -31,9 +31,10 @@ def start_monitoring_torch_compile(vllm_config: VllmConfig):
 def end_monitoring_torch_compile(vllm_config: VllmConfig):
     compilation_config: CompilationConfig = vllm_config.compilation_config
     if compilation_config.mode == CompilationMode.VLLM_COMPILE:
+        total_elapsed = time.time() - torch_compile_start_time
         logger.info_once(
-            "torch.compile takes %.2f s in total",
-            compilation_config.compilation_time,
+            "torch.compile total time: %.2f s",
+            total_elapsed,
```

Comment:
what's the difference between these two times? Why compilation_config.compilation_time doesn't work?